src/asmcnc/comms/serial_connection.py

```diff
@@ -39,10 +39,38 @@ class MachineState(Enum):
 
 
 class SerialConnection(EventDispatcher):
+    setting_0 = NumericProperty(0.0)
```

Comment:
instantiate the values with -1.0
Otherwise we won't get the event if the value is actually 0 as event are fired on_change. E.g. if $22 (homing cycle, needs to be 1) is set to 0 for whatever reason why won't get the event and can't react. (I probably should do this for $100 - $102 as well...)